'base64.h' file not found in MGLUtils.cpp

After upgrading to `0.7.0-rc.2` I am receiving the following error when building iOS through `eas`:

```
[RUN_FASTLANE] › Compiling react-native-quick-crypto Pods/react-native-quick-crypto » MGLUtils.cpp
[RUN_FASTLANE] 
❌  (node_modules/react-native-quick-crypto/cpp/Utils/MGLUtils.cpp:9:10)

   7 | #include <string>
   8 | 
>  9 | #include "base64.h"
     |          ^ 'base64.h' file not found
  10 | 
  11 | #ifdef ANDROID
  12 | #include "JSIUtils/MGLJSIMacros.h"
```

Here are the relevant package versions in my `package.json`:
```
"expo": "~50.0.14",
"react-native": "0.73.6",
"react-native-quick-base64": "^2.1.2",
"react-native-quick-crypto": "0.7.0-rc.2",
```

> I get that, and I am suggesting use the release candidates, not downgrade. But why does Expo need the extra `../../` on your include_directories paths? If dependencies are installed properly, that shouldn't be the case. That's what I don't understand. But I don't use Expo.

That was @pehkay's solution above for Android. That is not a solution I can implement though, since it is a managed Expo application so the native ios and android folders and code are not available to edit. I had not yet tested Android when creating this issue, but after just trying to build it I am getting the same error:

```
[RUN_GRADLEW]   /private/var/folders/t8/qqvt248d71l8td83wd3ksr8c0000gp/T/eas-build-local-nodejs/97c49f93-05be-44bc-9f82-f3aa3d0b1ddc/build/node_modules/react-native-quick-crypto/cpp/Utils/MGLUtils.cpp:9:10: fatal error: 'base64.h' file not found
[RUN_GRADLEW]   #include "base64.h"
[RUN_GRADLEW]            ^~~~~~~~~~
[RUN_GRADLEW]   1 error generated.
```

--- a/android/CMakeLists.txt
+++ b/android/CMakeLists.txt
@@ -5,7 +5,6 @@ set(PACKAGE_NAME "reactnativequickcrypto")
 set(BUILD_DIR ${CMAKE_SOURCE_DIR}/build)
 set(CMAKE_VERBOSE_MAKEFILE ON)
 set(CMAKE_CXX_STANDARD 17)
-set(NODE_MODULES_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../node_modules")
 
 # set (CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -g")
 # set (CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -g")
@@ -48,7 +47,7 @@ add_library(
 )
 
 include_directories(
-  "../node_modules/react-native-quick-base64/cpp"
+  "${NODE_MODULES_DIR}/react-native-quick-base64/cpp"
 )
 
 target_include_directories(

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -112,7 +112,7 @@ android {
     externalNativeBuild {
       cmake {
         cppFlags "-O2 -frtti -fexceptions -Wall -Wno-unused-variable -fstack-protector-all"
-        arguments '-DANDROID_STL=c++_shared' , "-DNODE_MODULES_DIR=${nodeModules}",
+        arguments "-DANDROID_STL=c++_shared", "-DNODE_MODULES_DIR=${nodeModules}"
         abiFilters (*reactNativeArchitectures())
       }
     }

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -48,22 +48,22 @@ def reactNativeArchitectures() {
   return value ? value.split(",") : ["armeabi-v7a", "x86", "x86_64", "arm64-v8a"]
 }
 
-// static def findNodeModules(baseDir) {
-//   def basePath = baseDir.toPath().normalize()
-//   // Node's module resolution algorithm searches up to the root directory,
-//   // after which the base path will be null
-//   while (basePath) {
-//     def nodeModulesPath = Paths.get(basePath.toString(), "node_modules")
-//     def reactNativePath = Paths.get(nodeModulesPath.toString(), "react-native")
-//     if (nodeModulesPath.toFile().exists() && reactNativePath.toFile().exists()) {
-//       return nodeModulesPath.toString()
-//     }
-//     basePath = basePath.getParent()
-//   }
-//   throw new GradleException("react-native-quick-crypto: Failed to find node_modules/ path!")
-// }
+static def findNodeModules(baseDir) {
+  def basePath = baseDir.toPath().normalize()
+  // Node's module resolution algorithm searches up to the root directory,
+  // after which the base path will be null
+  while (basePath) {
+    def nodeModulesPath = Paths.get(basePath.toString(), "node_modules")
+    def reactNativePath = Paths.get(nodeModulesPath.toString(), "react-native")
+    if (nodeModulesPath.toFile().exists() && reactNativePath.toFile().exists()) {
+      return nodeModulesPath.toString()
+    }
+    basePath = basePath.getParent()
+  }
+  throw new GradleException("react-native-quick-crypto: Failed to find node_modules/ path!")
+}
 
-// def nodeModules = findNodeModules(projectDir)
+def nodeModules = findNodeModules(projectDir)
 
 repositories {
   google()
@@ -112,7 +112,7 @@ android {
     externalNativeBuild {
       cmake {
         cppFlags "-O2 -frtti -fexceptions -Wall -Wno-unused-variable -fstack-protector-all"
-        arguments '-DANDROID_STL=c++_shared' //, "-DNODE_MODULES_DIR=${nodeModules}",
+        arguments '-DANDROID_STL=c++_shared' , "-DNODE_MODULES_DIR=${nodeModules}",
         abiFilters (*reactNativeArchitectures())
       }
     }

--- a/example/src/testing/Tests/webcryptoTests/import_export.ts
+++ b/example/src/testing/Tests/webcryptoTests/import_export.ts
@@ -91,7 +91,7 @@ describe('subtle - importKey / exportKey', () => {
         await subtle.importKey('raw', 1, { name: 'PBKDF2' }, false, [
           'deriveBits',
         ]),
-      'invalid argument type "key"'
+      'Invalid argument type for "key". Need ArrayBuffer, KeyObject, CryptoKey, string'
     );
     await assertThrowsAsync(
       async () =>

